river_crossing/lib/river_crossing.ex

```diff
@@ -0,0 +1,33 @@
+defmodule RiverCrossing do
+  @type state :: :in_process | {:done, map()}
+
+  @spec start() :: any()
+  def(start()) do
```

Comment:
Больше скобок богу скобок?
```suggestion
  def start do
```